pygeoapi/api.py

```diff
@@ -680,31 +680,37 @@ def describe_processes(self, headers, args, process=None):
             LOGGER.error(exception)
             return headers_, 400, json.dumps(exception)
 
-        processes_config = self.config['processes']
+        processes_config = self.config.get('processes', {})
 
-        if process is not None:
-            if process not in processes_config.keys():
-                exception = {
-                    'code': 'NotFound',
-                    'description': 'identifier not found'
-                }
-                LOGGER.error(exception)
-                return headers_, 404, json.dumps(exception)
+        if processes_config:
```

Comment:
Notice that `processes_config` will always have a (`dict`) value now. First thought  the `else` case would never be reached. But then again line 685 will evaluate to `True` if `processes_config` is not `None` AND non-empty [as per Python rules](https://docs.python.org/2/library/stdtypes.html#truth-value-testing). So no change required.